Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -547,6 +547,7 @@ subscripts : subscript (whiteSpace? COMMA whiteSpace? subscript)*;
 subscript : (expression whiteSpace TO whiteSpace)? expression;
 
 unrestrictedIdentifier : identifier | statementKeyword | markerKeyword;
+legalLabelIdentifier : { !(new[]{DOEVENTS,END,CLOSE,ELSE,LOOP,NEXT,RANDOMIZE,REM,RESUME,RETURN,STOP,WEND}).Contains(_input.La(1))}? identifier | markerKeyword;
```

Comment:
The illegalLabelIdentifiers could be maintained through a partial class' readonly  field. That would simplify the predicate. We are using native predicates anyways